### PR TITLE
Eve08

### PR DESCRIPTION
**Pull request by NotSpecial**
Eve 0.8 brings Cerberus 1.0, which requires lots of changes:

- Several properties have been renamed
- type checking is now handled differently
- validation rules require a schema in the docstring to validate the schema
  itself

This patch updates AMIV API to work with Cerberus 1.0.

### CODE REVIEW

File: amivapi/events/validation.py [hunk 1]
```diff
@@ -138,68 +145,81 @@ def _validate_email_signup_must_be_allowed(self, enabled, field, value):
...
-        """Retrieve time field from document or _original_document."""
+        """Retrieve time field from document or original document."""
         # Try to pick the value from document first, fall back to original
         time = self.document.get(fieldname)
         if time is None:
-            time = (self._original_document[fieldname]
-                    if self._original_document else None)
-
-        if time is None:
-            return None
```
Comment (cburchert) [inline amivapi/events/validation.py]: Why did you remove these two line? If time is now None here, this will crash the API as None does not have a replace function, which is still called in the next line.

File: amivapi/events/validation.py [hunk 2]
```diff
@@ -138,68 +145,81 @@ def _validate_email_signup_must_be_allowed(self, enabled, field, value):
...
+    def _validate_json_schema(self, enabled, field, value):
+        """Validate a json schema[1] string.
 
-        1.  Is it JSON?
+        1.  Is the string valid JSON?
         2.  Does it satisfy our restrictions for jsonschemas?
         3.  Is it a valid json-schema?
 
         Args:
-            field (string): field name.
```
[the remark this answers is not in the document]
Comment (NotSpecial) [reply amivapi/events/validation.py]: Oh yes, this is a mistake: At first, I tried to use the new cerberus types, which to not have `field` anymore, so I removed it. Later I decided that using a `string` type and additional validator works better with our online docs, so I changed the function, but forgot to update the docstring.

Will correct :)

File: amivapi/events/validation.py (hunk 1 above)
Comment (NotSpecial) [reply amivapi/events/validation.py]: I merged it into the return statement: `return time.replace(tzinfo=None) if time is not None else None`

Do you think this looks too confusing? Then I can change it back^^
